doc/did-method-spec.md

```diff
@@ -109,8 +112,8 @@ If the identifier is a `public-key-hex`:
 - it MUST be represented in compressed form (see https://en.bitcoin.it/wiki/Secp256k1)
 - the corresponding `blockchainAccountId` entry is also added to the default DID document, unless the `owner` has been
   changed to a different address.
-- all Read, Update, and Delete operations MUST be made using the corresponding `blockchainAccountId` and MUST originate from the correct `owner`
-  address.
+- all Read, Update, and Delete operations MUST be made using the corresponding `blockchainAccountId` and MUST originate
+  from the correct `owner` address.
```

Comment:
`controller` address? just want to ensure we have a consistent terminology.

Reply:
right.
`owner` is mentioned here because that is the name of the variable (and related events and methods) in the ERC1056 contract.

Does it make sense then to refer to it as `controller (owner) address` throughout the spec?

Reply:
I added a section specifically about this